scripts/us_bea/states_gdp/import_industry_data_and_gen_mcf.py

```diff
@@ -0,0 +1,168 @@
+            return date[:4] + "-" + qtr_month_map[date[5:]]
+        df['Quarter'] = df['Quarter'].apply(date_to_obs_date)
+
+        def convert_geoid(fips_code):
+            """Creates GeoId column. We get lucky that Data Commons's geoIds
+            equal US FIPS state codes.
+            """
+            fips_code = fips_code.replace('"', "")
+            fips_code = fips_code.replace(" ", "")
+            return "geoId/" + fips_code[:2]
+        df['GeoId'] = df['GeoFIPS'].apply(convert_geoid)
+
+        df = df[df['IndustryClassification'] != "..."]
+
+        def convert_industry_class(naics_code):
+            """Filters out aggregate NAICS codes and assigns them their Data
+            Commons codes.
+            """
+            if naics_code == "321,327-339":
+                return "JOLTS_320000"
+            if naics_code == "311-316,322-326":
+                return "JOLTS_340000"
+            return naics_code.replace("-", "_")
+        df['NAICS'] = df['IndustryClassification'].apply(convert_industry_class)
+
+        self.df = df.drop(["GeoFIPS","IndustryClassification"], axis=1)
+
+    def download_data(self):
+        """Downloads zip, extracts the desired CSV, and puts it into a DF."""
+        # Open zip file from link.
+        resp = urlopen(self.ZIP_LINK)
+
+        # Read the file, interpret it as bytes, and create a ZipFile instance
+        # from it for easy handling.
+        zip_file = zipfile.ZipFile(io.BytesIO(resp.read()))
+
+        # Open the specific desired file (CSV) from the folder, and decode it.
+        # This results in a string representation of the file. Interpret that
+        # as a CSV, and read it into a DF.
+        data = zip_file.open(self.FILE).read().decode('utf-8')
+        data = list(csv.reader(data.splitlines()))
+        df = pd.DataFrame(data[1:], columns=data[0])
+        return df
+
+    def save_csv(self, prefix='states_industry_gdp'):
+        """Splits instance data frame into multiple CSV files, one per industry
+        code. This facilitates TMCF design.
+        """
+        for naics_code, naics_df in self.df.groupby("NAICS"):
+            naics_df.to_csv(prefix + f"_{naics_code}.csv")
+
+    def generate_MCF_TMCF(self):
+        """Generates MCF StatVars for each industry code."""
+        mcf_temp = ('Node: dcid:USSateQuarterlyIndustryGDP_NAICS_{naics}\n'
+                    'typeOf: dcs:StatisticalVariable\n'
+                    'populationType: dcs:EconomicActivity\n'
+                    'activitySource: dcs:GrossDomesticProduction\n'
+                    'measuredProperty: dcs:amount\n'
+                    'measurementQualifier: dcs:Nominal\n'
+                    'naics: dcid:NAICS/{naics}\n\n')
+        tmcf_temp = ('Node: E:states_industry_gdp_{naics}->E1\n'
+                     'typeOf: dcs:StatVarObservation\n'
+                     'variableMeasured: dcs:USSateQuarterlyIndustryGDP_NAICS_{naics}\n'
```

Comment:
```suggestion
                     'variableMeasured: dcs:USStateQuarterlyIndustryGDP_NAICS_{naics}\n'
```

Reply:
No lobger generate the tmcf file, so this is handled.